### DISCUSSION TITLE
Project Donations Announcement

### DISCUSSION THREAD

**Discussion by emabrey** [tenacityteam/tenacity-legacy#265] [Announcements]
Because many users have asked, I have setup an Open Source Collective based donation option. You can donate using the Github Sponsors feature and the money will be placed into the control of the Fiscal Host, which is the Open Source Collective. All expenditures will be public and require approval, that way donators know that their donations are being used appropriately.

The Open Source Collective for Tenacity is located [here](https://opencollective.com/tenacity).
Donations can be submitted there directly, or alternatively, you can [use the GitHub sponsors feature](https://github.com/sponsors/tenacityteam).

Some of our developers are also independently offering donation options that DO NOT go to the project, but rather directly to the individual, myself included. I want to clarify that if you donate via our individual profile pages ([mine](https://github.com/sponsors/emabrey), or [falktx](https://github.com/falktx), for example) that DOES NOT GO TO THE PROJECT. There is nothing untoward about offering individual donation options, of course, but I do not want there to be any confusion.

**Comment by SubOptimal** [comment]
Interesting how fast opinions are changed within two days.

![grafik](https://user-images.githubusercontent.com/653288/125687270-f3d4fdad-06c4-409e-8a75-dfce20908d84.png)

origin: https://github.com/tenacityteam/tenacity/discussions/139#discussioncomment-993004

**Reply by ajayyy** [reply]
GitHub just sends it to open-collective. You can also donate directly from open collective

**Reply by Semisol** [reply]
I would rather not use GH sponsors
But I don't know.

**Reply by emabrey** [reply]
GitHub sponsors literally just goes through to open collective. GitHub is not involved in anyway except for providing an additional UI for sending money to open collective. If you don't want to use GitHub, simply go directly to https://opencollective.com/tenacity/. GitHub holds the money for the time period between donation and the next time it becomes the 22nd of the month, then it is sent to the Open Collective account.

**Comment by falkTX** [comment]
So going to be real honest.

Accepting donations puts me in an uncomfortable position, because money is being exchanged. It is no longer a simple project "for fun" trying to move away from Muse Group and setting up an alternative.
With money involved, things get serious.

People will have expectations, and audacity being famous (compared to such other tools) makes it a target for people's hopes.
It also taints the kindness of the effort, because efforts could be made in pursuit of money/donations instead of willingly give time to building an audacity alternative.

Of course money is needed to survive and keep things running, but a release should have been made first.
Tenacity still needs to prove itself, before to trying to get money from people.
If it doesn't work out, people will have donated for a project that never properly left the ground.

It is, in my opinion, really inappropriate at this time.
For this reason I am distancing myself from the Tenacity team, but wish you all the best.

**Reply by purplesyringa** [reply]
> Tenacity still needs to prove itself, before to trying to get money from people. If it doesn't work out, people will have donated for a project that never properly left the ground.

Agreed; it's not like no one understands the risks though. I think someone has mentioned somewhere (I can't find it atm) that if Tenacity doesn't survive the donations will be returned to the sponsors, which is probably a good decision for now.

**Reply by emabrey** [reply]
That is your decision to make, but I don't think transparently taking donations that go to a non-individually controlled open source collective is a problem. People were already asking for the ability to donate, so I provided it to them. If people want to wait until after the first release I believe that would be prudent, but I also didn't want to tell people who were asking for a way to contribute that didn't involve code "No." I don't know why you thought Tenacity was "for fun", meaning I assume not a serious endeavor, but I wish you well. Once the project has been around for a longer period of time please consider coming back and contributing, and I hope you will be a user either way. Thank you for your comments.

**Reply by emabrey** [reply]
> 
> 
> > Tenacity still needs to prove itself, before to trying to get money from people. If it doesn't work out, people will have donated for a project that never properly left the ground.
> 
> Agreed; it's not like no one understands the risks though. I think someone has mentioned somewhere (I can't find it atm) that if Tenacity doesn't survive the donations will be returned to the sponsors, which is probably a good decision for now.

The GitHub sponsors page states:

"_Please note that upon the unlikely termination of the development of Tenacity, every effort will be made to return any unused funds to the donators or, failing that, the unused funds will be donated to another Open Source software project._"

**Comment by Songtech-0912** [comment]
I think that this step is an excellent step in the right direction. Sure, money is a very iffy thing for lots of people involved with free/libre software communities, but it shouldn't have to be. The GNU GPL (in all its variants) does NOT prohibit us from making commercial software, so long as we aren't making proprietary nonfree software. To be sure, I don't want to overly monetize Tenacity, but it should get at least some funding. It sets a precedent for other open-source projects, especially those struggling financially, to receive more monetary support!

Let's face it, we are always in a perilous position because of the pretty difficult past Tenacity has faced. In my experience, especially with another open-source software community ([Natron](https://github.com/NatronGitHub/Natron)), the lack of any sort of income puts an incredibly heavy pressure on the existing developers. In some cases, it is enough to sink even a very well established project. Especially consider the fact that right now, Tenacity is the literal manifestation of a lot of free software's ideals. We are going with a strong copyleft license (which will likely grow even stronger with time, see https://github.com/tenacityteam/tenacity/discussions/180), against the contemporary trend of being a permissively-licensed project in corporate control. This alone will make things hard enough that I think at least some monetary support is needed.

I apologize that this post doesn't have the non-biased attitude I usually try to maintain, but I hope it is nonetheless a noteworthy opinion to share.

**Comment by processor286** [comment]
I've donated to other floss projects through things like issuehunt before, so this seems reasonable to me. Donate, or do not donate. There is no obligation.

**Reply by fitojb** [reply]
Exactly!!

**Reply by Ratteler** [reply]
Can we just a Patreon going? I would join a Patreon.

**Reply by caughtquick** [reply]
> Can we just a Patreon going? I would join a Patreon.

Patreon takes a much larger amount of money than other options, isn't as transparent and has a questionable privacy policy.
